sparkle/platform/settings_help.py

```diff
@@ -75,7 +75,9 @@ class Settings:
     DEFAULT_general_penalty_multiplier = 10
     DEFAULT_general_extractor_cutoff_time = 60
 
-    DEFAULT_config_budget_per_run = 600
+    DEFAULT_config_wallclock_time = 2147483647
```

Comment:
Maybe keep this one to 600, otherwise if the user forgets to write down any limitations for the configurator it will run to infinity